app.py

```diff
@@ -5,15 +5,8 @@
 from werkzeug.exceptions import HTTPException
 from error_handling import CustomError, ValidationError, WrongIdError
 
-app = Flask(__name__)
-env = os.environ.get('FLASK_ENV', 'production')
-if env == 'development':
-    app.config.from_object('config.DevConfig')
-else:
-    os.environ['FLASK_ENV'] = 'production'
-    app.config.from_object('config.ProdConfig')
-
-with open(app.config['PATH_FOR_INITIAL_DATA'], "r") as read_file:
+path = os.environ.get('PATH_FOR_INITIAL_DATA', 'restaurants.json')
```

Comment:
by the way, now the whole app is initiated in `create_app()` so it might make sense to read the data in `create_app` too.
because what is happening now: 
when you write `import delivery_system.app` the code in `app.py` is executed and data is read while you might not wanted that. it can be called a side-effect.

Reply:
fixed